kube_resource_report/report.py

```diff
@@ -525,6 +535,37 @@ def resolve_application_ids(applications: dict, teams: dict, application_registr
             teams[team_id] = team
 
 
+def resolve_team_labels(applications: dict, teams: dict):
+    for app in applications.values():
+        if app["team"] != "":
+            team_id = app["team"]
+            app["team"] = team_id
```

Comment:
why do you assign this again when team_id was just assigned to `app["team"]`?

Reply:
probably a something I did differently before and overlooked afterwards, I will remove 541 and 542